keras_nlp/models/bert/bert_tokenizer.py

```diff
@@ -93,7 +99,12 @@ def set_vocabulary(self, vocabulary):
 
         if vocabulary is not None:
             # Check for necessary special tokens.
-            for token in [self.cls_token, self.pad_token, self.sep_token]:
+            for token in [
```

Comment:
I think with this change, we might want to introduce a `special_tokens` property to the base class, and move this check, that all special tokens are the vocabulary, to the base class. WDYT?

Reply:
sounds good. I have added special tokens check inside `set_vocabulary` function. take a look at the latest changes.